src/composables/useApi.ts

```diff
@@ -0,0 +1,63 @@
+// libraries
+import { Notify } from 'quasar';
+import { api } from '../boot/axios';
+import { i18n } from '../boot/i18n';
+
+// config
+import { rideToWorkByBikeConfig } from '../boot/global_vars';
+const { apiVersion } = rideToWorkByBikeConfig;
+
+// types
+import type { Method } from 'axios';
+
+interface ApiResponse<T> {
+  data: T | null;
+}
+
+export const useApi = () => {
+  const apiFetch = async <T>({
+    endpoint,
+    payload,
+    translationKey,
+    method = 'get',
```

Comment:
`headers` param should be here with default API version value, because in some URL endpoints calling we will want override API version.

Reply:
Updated. Made `apiVersion` a parameter with default.